vendor/assets/stylesheets/components/_chips.scss

```diff
@@ -0,0 +1,27 @@
+.chip {
+  display: inline-block;
```

Comment:
Properties should be ordered background-color, border-radius, color, display, font-size, font-weight, height, line-height, padding